PubNub/Core/PubNub+Publish.m

```diff
@@ -88,6 +89,10 @@ - (void)  publish:(id)message toChannel:(NSString *)channel
     [self publish:message toChannel:channel compressed:NO withCompletion:block];
 }
 
+- (void)publish:(id)message toChannel:(NSString *)channel withMetadata:(NSDictionary *)metadata withCompletion:(PNPublishCompletionBlock)block {
+    [self publish:message toChannel:channel mobilePushPayload:nil storeInHistory:YES compressed:NO withMetadata:metadata withCompletion:block];
```

Comment:
@jzucker2 All methods in project follow same formatting style: there is one empty line right after method signature.